### ISSUE TITLE
Improved character names

### ISSUE THREAD

**Issue by adamkiss** [adamkiss/alfred-utf#3]
As <kbd>⌘</kbd> shows, some names (`places of interest sign`) aren't really what people (me) might be looking for (`command`).

Look through original py2 workflow - https://github.com/bevesce/unicode-symbols-search - and unicode definition - https://www.unicode.org/Public/UCD/latest/ucd/NamesList.txt for alternative names, and think about how to best include them.

For the POI sign I just slapped ", command" at the end of the name, but some have multiple alternative names, and it doesn't make sense to make the subtitle 300 characters long/

**Comment by vitorgalvao** [comment]
Other suggestions:

* `↩` (Leftwards Arrow With Hook) matching “return”. That’s how it is read by screen readers.
* ⌃ (Up Arrowhead) matching “control” and “ctrl”.

**Comment by adamkiss** [comment]
@vitorgalvao yes. I'm working on a couple of improvements, and looking at merging alternative names from unicode. 

I'll need to hard code the 'ctrl', but that should be the only exception.

**Comment by vitorgalvao** [comment]
> I'll need to hard code the 'ctrl', but that should be the only exception.

In that case, I’d say only “control” is enough. “ctrl” is a small convenience which isn’t worth complicating over.

**Comment by adamkiss** [comment]
Thinking about the arrow, there's a difference between arrow with hook and proper return symbol:

```
↩ - leftwards arrow with hook
⏎ - return symbol
⮐ - return left
```

I think I might need to add the return manually as well.

ctrl isn't a problem either, I'm mixing two unicode sources anyway :D

**Comment by vitorgalvao** [comment]
> Thinking about the arrow, there's a difference between arrow with hook and proper return symbol

Correct. I used to use the “return symbol” as well, until I passed my text through macOS’ VoiceOver. If you turn it on and select the following text:

Press ↩
Press ⏎
Press ⮐

It reads the first as “return”, the second as “return symbol”, and the last one isn’t read at all. That’s likely why the first one is the one that is transformed to an emoji.
